.claude/scripts/fetch_polarion.py

```
@@ -0,0 +1,341 @@
+        idx = sys.argv.index('--project')
+        if idx + 1 < len(sys.argv):
+            project = sys.argv[idx + 1]
+
+    # Fetch test case
+    tc_data = fetch_test_case(tc_id, project)
+
+    # Output JSON to stdout
+    print(json.dumps(tc_data, indent=2))
+
+    # Write text format for /automate-test
+    output_dir = '/tmp/test-specs'
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/test-specs"

(S108)